### PR TITLE
add different save ows name strategies

### PR DESCRIPTION
**Pull request by rduivenvoorde**
became a little more complex then I hoped, please review and test

one bug fixed:
https://github.com/geopython/MetaSearch/blob/master/plugin/MetaSearch/dialogs/maindialog.py#L652
saving a Setting with a key part like 'OGC:WMS/OGC:WMTS' silently failed. 
So I changed it to a simpler: "wms from MetaSearch"

Pressing the enter button now closed the dialog with me (as I had earlier myself with another plugin). Fixed with line 105

### CODE REVIEW

File: plugin/MetaSearch/dialogs/maindialog.py [hunk 1]
```diff
@@ -133,6 +138,12 @@ def manageGui(self):
 
         self.reset_buttons()
 
+        # get latest connection save strategy from settings and set it in dialog
+        saved = self.settings.value(
+            '/metasearch/ows_save_strategy',
```
Comment (tomkralidis) [inline plugin/MetaSearch/dialogs/maindialog.py]: I see we introduce the `/metasearch/` namespace in settings.  Should we harmonize this across the plugin and change `/CSWClient/` to `/MetaSearch/`? If yes, let's open another ticket on this specifically.

**Comment by tomkralidis** [comment]
@rduivenvoorde I get an error when trying build the .ui file (using pyuic4 from OSGeo4W):

pyuic4 -o maindialog.py maindialog.ui

"An unexpected error occurred."

File: plugin/MetaSearch/dialogs/maindialog.py (hunk 1 above)
Comment (rduivenvoorde) [inline plugin/MetaSearch/dialogs/maindialog.py]: I think we should have our own 'namespace' there?
So yes: plz open a ticket.

I do not like capitals in url's and keys, thats why I make metasearch of it, but I'm happy too with MetaSearch :-)

Comment (tomkralidis) [inline plugin/MetaSearch/dialogs/maindialog.py]: ok we can take care of this after in #29

Comment (tomkralidis) [inline plugin/MetaSearch/dialogs/maindialog.py]: Fixed by @alexbruy. So this can now be called MetaSearch.

**Comment by rduivenvoorde** [comment]
@tomkralidis plz try again

**Pull request merged**